os/tools/mkbootparam.py

```diff
@@ -37,10 +43,10 @@
 # | Checksum | BP Version | BP Format Version | Active Idx | First Address | Second Address |
 # | (4bytes) |  (4bytes)  |      (4bytes)     |   (1byte)  |    (4bytes)   |    (4bytes)    |
 # +------------------------------------------------------------------------------------------
-# ----------------------------------------------------------------------------------------------------------------+
-# | App Count | App1 Name  | App1 Active Idx | App2 Name  | App2 Active Idx |     App# ...   |      Reserved      |
-# |  (1byte)  | (16 bytes) |   (1 bytes)     | (16 bytes) |    (1 bytes)    |  (## bytes)    |(8Kbytes - 21bytes) |
-# ----------------------------------------------------------------------------------------------------------------+
+# ----------------------------------------------------------------------------------------------------------------------+
+# | App Count | App1 Name  | App1 Active Idx | App2 Name  | App2 Active Idx |     App# ...   |         Reserved         |
+# |  (1byte)  | (16 bytes) |   (1 bytes)     | (16 bytes) |    (1 bytes)    |  (## bytes)    |(8Kbytes - (21 + @)bytes) |
```

Comment:
There are two BP parts, BP1 and BP2 so that "Reserved" should be up to 4KB, not 8KB. Could you confirm this?

Reply:
I wrote 8K because this script generates initial boot parameter binary (-> puts BP data to first BP, BP1 and clear second BP, BP2.)
I updated 4K as you said and added additional comment to reduce user confusion. Please check again.